source/io.c

```diff
@@ -453,7 +523,8 @@ PyObject *aws_py_client_tls_ctx_new(PyObject *self, PyObject *args) {
             goto ctx_options_failure;
         }
     }
-    if (ca_buffer_len > 0) {
+
+    if (ca_buffer) {
```

Comment:
460 uses 
if (certificate_buffer != NULL) 
as the check vs. if(ca_buffer) here.

Reply:
yeah, our code is pretty inconsistent on explicit comparisons `if (x != NULL)` 
vs
implicit comparisons `if (x)`

I was just trying to match whatever code was nearest
but I'll make the function more consistent, and favor explicit comparisons